docs/api/templates.md

```diff
@@ -175,10 +175,17 @@ Status Code **200**
 | `» id`                               | string(uuid)                      | false    |              |                                            |
 | `» name`                             | string                            | false    |              |                                            |
 | `» organization_id`                  | string(uuid)                      | false    |              |                                            |
-| `» provisioner`                      | string                            | false    |              |                                            |
+| `» provisioner`                      | `codersdk.ProvisionerType`        | false    |              |                                            |
 | `» updated_at`                       | string(date-time)                 | false    |              |                                            |
 | `» workspace_owner_count`            | integer                           | false    |              |                                            |
 
+#### Enumerated Values
+
+| Property      | Value       |
+| ------------- | ----------- |
+| `provisioner` | `echo`      |
```

Comment:
Let's not expose the `echo` provider, it's only used by tests. Can we remove it from the enum or add a replace to swaggo to just show it as a string?

Reply:
Fixed. It shows only `terraform` now.